tests/conftest.py

```diff
@@ -101,6 +101,7 @@ def endpoint_fixture(
     port_forward_payload: list[dict[str, Any]],
     site_payload: list[dict[str, Any]],
     system_information_payload: list[dict[str, Any]],
+    traffic_rule_payload: list[dict[str, Any]],
```

Comment:
The error message is something related to the two fixtures. One reason could be that both methods are defined as `def endpoint_fixture`. I still want you to make traffic_rule_payload work with the original endpoint_fixture and remove _mock_traffic_rule_endpoint

Reply:
I've fixed this and made some other changes to some tests (they were not yet working with TypedApiResponse but with raw dicts). But now I have one of my test failing :confounded:
Almost there...

Reply:
I've got everything up and running. Had to make some modifications to some other tests, since process_raw now processes a TypedApiResponse (and no longer a 'raw' dict).